microsoft-edge/webview2/release-notes.md

```diff
@@ -32,6 +32,106 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 
 To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
 
+<!-- ====================================================================== -->
+## 1.0.1418.22
+
+Release Date: October 31, 2022 
+
+[NuGet package for WebView2 SDK 22](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1418.22)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 107.0.1418.22 or higher.
+
+### General 
+
+This WebView2 SDK release has the same bug fixes that are in WebView2 SDK 1.0.1466-prerelease. See Bug fixes in the following section.
+
+---
+
+
+<!-- ====================================================================== -->
+## 1.0.1466-prerelease
+
+Release Date: October 31, 2022
+
+[NuGet package for WebView2 SDK 1.0.1466-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1466-prerelease)
+
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 109.0.1466.0 or higher.
+
+### General
+
+#### Experimental features
+
+* Added support to create a shared memory based buffer with the specified size:
+    *  `close`
+    *  `get_Buffer`
+    *  `get_FileMappingHandle`
+    *  `get_Size`
+    *  `OpenStream`  
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2SharedBuffer Class](/dotnet/api/microsoft.web.webview2.core.corewebview2sharedbuffer?view&preserve-view=true=webview2-dotnet-1.0.1466-prerelease)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2SharedBuffer Class](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalsharedbuffer?view&preserve-view=true=webview2-1.0.1466-prerelease)

```

Comment:
```suggestion
* [CoreWebView2SharedBuffer Class](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalsharedbuffer?view=webview2-1.0.1466-prerelease&preserve-view=true)
```
Expect fixed link to =
https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2experimentalsharedbuffer?view=webview2-1.0.1466-prerelease&preserve-view=true